docs/dev/disk-image-install.md

```diff
@@ -0,0 +1,22 @@
+# Assisted Installation With a Disk Image
+
+You can use the Assisted Installer to install to install directory to the existing root filesystem of a disk image.
+This should be done for platforms that cannot take the discovery ISO as boot media, and also cannot use iPXE.
+
+## Available Disk Images
+
+You can download disk images for use with this process at https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos
+An example for installing OCP 4.17 with libvirt would be https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.17/4.17.2/rhcos-4.17.2-x86_64-qemu.x86_64.qcow2.gz
+
+## Discovery and Installation
+
+You must write the Assisted Installer discovery ignition to the disk on first boot in order to install in this way.
+You can do this using whatever metadata service is available for your target platform.
+
+You can download the discovery ignition assisted-service REST API.
+A presigned URL is published on the InfraEnv status (`.status.bootArtifacts.discoveryIgnitionURL`) if using the kubernetes APIs.
+Otherwise you can download it using the endpoint `/v2/infra-envs/{infra_env_id}/downloads/files` with the query parameter `file_name=discovery.ign`
+
+Once the Assisted Installer agent is running, discovery will proceed as normal outside of some disabled validations (specifically the disk speed check which writes directly to the target install device).
```

Comment:
Sorry, I'm not familiar with this method. Do you need to run something after the ignition is written to disk?

Reply:
Run what? The validation? We don't run it. That's what I'm calling out here.

Reply:
On first boot of the host the discovery ignition is written to disk. This contains the service to run the agent.

When you start a host in this way the disk it is created with already contains rhcos and it will look to pull ignition over some metadata service that is specific to the platform. So this all happens before we "install" anything (just like with the ISO).

Reply:
Oh! Sorry, I didn't know it started like the live iso (on first boot [duh 🤦‍♀️] ). Thanks for the clarification!